app/controllers/sufia/resource_sync_controller.rb

```diff
@@ -1,26 +1,14 @@
 class Sufia::ResourceSyncController < ApplicationController
   def source_description
-    # Caching based on host, for multitenancy support
-    body = Rails.cache.fetch("source_description_#{request.host}", expires_in: 1.week) do
-      build_source_description
-    end
-    render body: body, content_type: 'application/xml'
+    render_from_cache_as_xml(build_source_description)
```

Comment:
This is obviating the cache.  It's running the build_source_description even if it's available in the cache.